_sass/leaderboard.scss

```
@@ -0,0 +1,542 @@
+}
+
+.podium-username {
+    font-size: 2.2rem;
+    font-weight: 700;
+    line-height: 1.1;
+    text-align: center;
+    color: #00B39F;
+    transition: color 0.2s ease;
+    width: 100%;
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)